### PR TITLE
Display review data

### PR DESCRIPTION
**Pull request by ghickman**
Add per-Release and per-ReleaseFile metadata

**Release**
We're just saving raw data to the metadata for Release currently so this isn't the most useful but we can at least show that we've captured parts of it?

**ReleaseFile**

### CODE REVIEW

File: assets/src/scripts/outputs-viewer/components/Metadata/Metadata.jsx [hunk 1]
```diff
@@ -46,6 +53,19 @@ function Metadata({ fileDate, fileName, fileSize, fileUrl }) {
         )}
         <li className="list-inline-item spacer">{prettyFileSize(fileSize)}</li>
       </ul>
+      <div>
+        <div className="mb-3">
+          <strong>What does this file contain?</strong>
```
Comment (bloodearnest) [inline assets/src/scripts/outputs-viewer/components/Metadata/Metadata.jsx:58]: This should be the key value of the dict

File: assets/src/scripts/outputs-viewer/components/Metadata/Metadata.jsx [hunk 2]
```diff
@@ -46,6 +53,19 @@ function Metadata({ fileDate, fileName, fileSize, fileUrl }) {
...
+          <div>{context}</div>
+        </div>
+
+        <div>
+          <strong>
+            How has the author ensured that this file is non-disclosive?
```
Comment (bloodearnest) [inline assets/src/scripts/outputs-viewer/components/Metadata/Metadata.jsx:64]: Same as above.

File: assets/src/scripts/outputs-viewer/App.jsx [hunk 3]
```diff
@@ -137,6 +137,8 @@ function App({
               {isReviewEdit ? <FileReview selectedFile={selectedFile} /> : null}
               <Card>
                 <Metadata
+                  context={selectedFile.metadata.context}
```
Comment (bloodearnest) [inline assets/src/scripts/outputs-viewer/App.jsx:140]: This should be just `metadata=metadata`

The jsx component should iterate through the key/value pairs.

To match the current document, we'd expect a dict like:

````
{
    "Context": "user input context",
    "Controls": "user input controls",
}
```

**Review by bloodearnest — CHANGES_REQUESTED** [review]

**Comment by ghickman** [comment]
Updated to show what it looks like.  I'm not sold on using the keys for the labels as that ties the display to the labels of the form, but I've updated it to that so we can see what it looks like for the demo.

**Comment by bloodearnest** [comment]
The keys have no semantic meaning - the can be the whole text of the form label if they want.

In fact, a dict of labels: values is a much better terminology

**Review by bloodearnest — APPROVED** [review]

**Pull request merged**